inc/infoutils.php

```diff
@@ -76,16 +76,32 @@ function getVersionData(){
         $version['type'] = 'Git';
         $version['date'] = 'unknown';
 
-        if ($date = shell_exec("git log -1 --pretty=format:'%cd' --date=short")) {
+        if (function_exists('shell_exec')) {
```

Comment:
now the fallback is never reached as long as the shell_exec function exists, even when git isn't installed or callable

Reply:
Right, I'll fix that. Thanks for the review !

Reply:
Should be fixed by b9f5205a8e8b015cdcdfbfbac335698045695f12